apps/researcher/src/lib/community.ts

```diff
@@ -28,8 +31,58 @@ export async function getMemberships(
   });
 }
 
-export function getAllCommunities(): Promise<Community[]> {
-  return clerkClient.organizations.getOrganizationList();
+export enum CommunitySortBy {
+  NameAsc = 'nameAsc',
+  NameDesc = 'nameDesc',
+  CreatedAtDesc = 'createdAtDesc',
+  MembershipCountDesc = 'membershipCountDesc',
+}
+
+export const defaultCommunitySortBy: CommunitySortBy =
+  CommunitySortBy.CreatedAtDesc;
+
+export function sortCommunities(
+  communities: Community[],
+  sortBy: CommunitySortBy
+) {
+  return [...communities].sort((a, b) => {
```

Comment:
Perhaps also add a comment here, explaining why we're not sorting by `MembershipCountDesc`?